### ISSUE TITLE
Forbid `from-import` for meaningless names

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#814]
# Rule request

## Thesis

Some packages designed to be imported only as module, and functions inside have name that completes module name.

Bad:

```python
from attr import s
from json import load, dumps
from tomlkit import parse
from schemathesis import from_file
```

## Reasoning

When you see `datetime(...)` in the code, you know that it's from `datetime`. When you see `BaseView` in a Django project, you know that it's from Django framework. When you see `loads`, it can be anything: yaml, toml, json, custom config.

## Proposed solution

Consider the next blacklist:

+ `load`, `loads`
+ `dump`, `dumps`
+ `parse`
+ PyYAML specific: `safe_load`, `safe_dump`, `load_all`, `dump_all`, `safe_load_all`, `safe_dump_all`
+ one-letter names
+ Anything that starts from `from_` or `to_`

**Comment by sobolevn** [comment]
Awesome!

**Comment by lensvol** [comment]
Can I take it?

**Comment by sobolevn** [comment]
Sure! Thanks a lot!

**Comment by lensvol** [comment]
I assume this new violation goes under "Best practices"?

**Comment by sobolevn** [comment]
I will say that it is a consistency rule.
